src/main/main.ts

```diff
@@ -10,9 +11,13 @@ const DEFAULT_TODOLIST_IDENTIFIER: TodolistIdentifier = {
 
 /**
  * Main entry point for the Onestop to Basecamp Integration that contains the core logic for
- * adding/updating/deleting Basecamp Todos based on the rows on the Onestop
+ * adding/updating/deleting Basecamp Todos based on the rows on the Onestop as well as a function to
+ * create a menu item to give users the ability to manually trigger the import process. 
  */
 export function importOnestopToBasecamp(): void {
+
+    updateManualTriggerMenuUiOnOneStop();
```

Comment:
question - so each time this function is called, the menu needs to be updated?

Reply:
Unless we are changing the menu all the time, we probably don't need to update it each time. We can just keep it as a manually  function from the AppScript editor

Reply:
the menu should be static, at least with our use case. if we write new code that adds a new menu option though, does that mean we'd need to run the function manually to update the menu? ideally we never have to call anything manually and the menu should reflect what we decide in the code

Reply:
maybe we'd have a function in menu.ts with a comment saying it needs to be run whenever something new is added?

Reply:
If we expect the menu to be mostly static then I think we can take it out from both the onOpen and the importOneStopToBasecamp functions and only run it manually via the AppScript editor. 


> the menu should be static, at least with our use case. if we write new code that adds a new menu option though, does that mean we'd need to run the function manually to update the menu? ideally we never have to call anything manually and the menu should reflect what we decide in the code

If we expect the menu to be mostly static then I think we can take it out from both the onOpen and the importOneStopToBasecamp functions and only run it manually via the AppScript editor.

Reply:
> maybe we'd have a function in menu.ts with a comment saying it needs to be run whenever something new is added?

Sure I can add that comment